src/pages/ContentScripts/features/oss-gpt/view.tsx

```diff
@@ -105,6 +145,11 @@ const View = ({ theme, currentRepo, currentDocsName }: Props): JSX.Element => {
     return () => {
       // Later, you can stop observing
       observer.disconnect();
+
+      if ($('.rcw-conversation-resizer')) {
+        $('.rcw-conversation-resizer').off('mousedown');
+        $(document).off('mousemove mouseup');
```

Comment:
Will `.off` remove all listeners (including GitHub own listeners) for mousemove and mouseup?

Maybe you can move the listeners more accurately.